[2023-09-05T10:13:19.531+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ganza_dag_v2.second_task scheduled__2023-08-30T00:00:00+00:00 [queued]>
[2023-09-05T10:13:19.571+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ganza_dag_v2.second_task scheduled__2023-08-30T00:00:00+00:00 [queued]>
[2023-09-05T10:13:19.577+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-05T10:13:19.631+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2023-08-30 00:00:00+00:00
[2023-09-05T10:13:19.646+0000] {standard_task_runner.py:57} INFO - Started process 1544 to run task
[2023-09-05T10:13:19.656+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ganza_dag_v2', 'second_task', 'scheduled__2023-08-30T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpdw8k1mh7']
[2023-09-05T10:13:19.661+0000] {standard_task_runner.py:85} INFO - Job 28: Subtask second_task
[2023-09-05T10:13:19.805+0000] {task_command.py:415} INFO - Running <TaskInstance: ganza_dag_v2.second_task scheduled__2023-08-30T00:00:00+00:00 [running]> on host 2729c554a115
[2023-09-05T10:13:20.088+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ganza_m' AIRFLOW_CTX_DAG_ID='ganza_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-30T00:00:00+00:00'
[2023-09-05T10:13:20.090+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-05T10:13:20.091+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hello, this is a second task2 and it will run after task1 has runned']
[2023-09-05T10:13:20.141+0000] {subprocess.py:86} INFO - Output:
[2023-09-05T10:13:20.145+0000] {subprocess.py:93} INFO - hello, this is a second task2 and it will run after task1 has runned
[2023-09-05T10:13:20.160+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-05T10:13:20.299+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ganza_dag_v2, task_id=second_task, execution_date=20230830T000000, start_date=20230905T101319, end_date=20230905T101320
[2023-09-05T10:13:20.385+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-05T10:13:20.507+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
